gen/definitions/standard_acl.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: Standard ACL
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/standardaccesslists
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: true
+    example: stdacl1
+  - model_name: description
+    type: String
+    description: Optional user-created description.
+    example: "My standard ACL"
+  - model_name: entries
+    type: List
+    description: Ordered list of ACL's entries.
+    ordered_list: true
+    mandatory: true
+    min_list: 1
+    max_list: 1000
+    attributes:
+      - model_name: action
+        type: String
+        description: "Indicates the redistribution access: PERMIT or DENY."
```

Comment:
Is that the description from the openapi spec? Are ACLs only used for redistribution?

Reply:
I haven't been able to pinpoint what are they used for, just feature parity. The desc is straight from the OpenAPI spec.

Reply:
```
      "IStandardACLEntry": {
        "required": [
          "action",
          "networks"
        ],
        "type": "object",
        "properties": {
          "action": {
            "type": "string",
            "description": "Indicate the redistribution access: PERMIT or DENY",
            "enum": [
              "PERMIT",
              "DENY"
            ]
          },
          "networks": {
            "type": "object",
            "description": "The network objects/groups assoicated with this object",
            "$ref": "#/components/schemas/INetworkObjectsContainer"
          }
        },
        "description": "Standard Access List entry"
      },